pkg/publish/kind/write.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2020 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kind
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	v1 "github.com/google/go-containerregistry/pkg/v1"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
+
+	"sigs.k8s.io/kind/pkg/cluster"
+	"sigs.k8s.io/kind/pkg/cluster/nodes"
+)
+
+// Tag adds a tag to an already existent image.
+func Tag(src, dest name.Tag) error {
+	return onEachNode(func(n nodes.Node) error {
+		cmd := n.Command("ctr", "--namespace=k8s.io", "images", "tag", "--force", src.String(), dest.String())
```

Comment:
I'm not super familiar with kind -- is this namespace flag somethign that would need to be configurable? Similar question re: `cluster.DefaultName`on line 76.

I understand this is a PoC, just curious how much config stuff we'll need to plumb through here for a final version.

Reply:
This namespace is hardcoded in kind too.

In theory, `kind` can have different cluster names and you could also potentially choose which nodes you load images on. I opted for the default cluster name here and opted to just publish to all nodes.